releases/v19.1.0-rc.2.md

```diff
@@ -0,0 +1,89 @@
+---
+title: What&#39;s New in v19.1.0-rc.2
+toc: true
+summary: Additions and changes in CockroachDB version v19.1.0-rc.2 since version v19.1.0-rc.1
+---
+
+## April 8, 2019
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.1.0-rc.2.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.1.0-rc.2.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.1.0-rc.2.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.1.0-rc.2.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v19.1.0-rc.2
+~~~
+
+### SQL language changes
+
+- Added `kv.bulk_io_write.concurrent_addsstable_requests` [cluster setting](../v19.1/cluster-settings.html), which limits the number of SSTables that can be added concurrently during bulk operations. [#36444][#36444]
```

Comment:
Might go with "Added the xxx setting" to match the other items

Reply:
Done.